win32/tests/job_object_helper_ut/job_object_helper_ut.c

```diff
@@ -0,0 +1,238 @@
+// Copyright(C) Microsoft Corporation.All rights reserved.
+
+
+#include <stdlib.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "testrunnerswitcher.h"
+#include "windows.h"
```

Comment:
We usually put windows.h before other of ours includes.
Typical order:
- C standard includes
- OS includes 
- our dependencies includes (usually lowest to highest)
- our current code includes.

Makes it easier to look for them (not that it won't work)